src/app/routes/utils/overrideRendererOnTest/index.js

```diff
@@ -0,0 +1,21 @@
+import Url from 'url-parse';
+import isLive from '#lib/utilities/isLive';
+
+const getQueryString = query => {
+  const params = Object.keys(query).map(key => `${key}=${query[key]}`);
+  return params.join('&');
+};
+
+export const addOverrideQuery = path => {
+  const url = new Url(path, true);
+  const { query } = url;
+  const searchString = getQueryString({ ...query, renderer_env: 'live' });
+  const pathName = path ? path.split('?')[0] : path;
+
+  return `${pathName}?${searchString}`;
```

Comment:
This can be simplifed a bit, by letting the `url` object do the heavy lifting for you:

```
  const overrideQuery = { ...query, renderer_env: 'live' };
  url.set('query', overrideQuery);
  return url.href;
```

Reply:
See https://www.npmjs.com/package/url-parse#urlsetkey-value